canvasapi/course.py

```diff
@@ -2347,6 +2347,80 @@ def list_blueprint_subscriptions(self, **kwargs):
             kwargs=combine_kwargs(**kwargs)
         )
 
+    def list_content_exports(self, **kwargs):
```

Comment:
To be consistent with other methods, please change this method name to `get_content_exports`. Do the same on the `User` and `Group` classes.